wpimath/src/main/java/edu/wpi/first/math/Matrix.java

```diff
@@ -707,8 +707,8 @@ public boolean isEqual(Matrix<?, ?> other, double tolerance) {
   /**
    * Performs an inplace Cholesky rank update (or downdate).
    *
-   * <p>If this matrix contains L where A = LL<sup>&top;</sup> before the update, it will contain L
-   * where LL<sup>&top;</sup> = A + &sigma;vv<sup>&top;</sup> after the update.
+   * <p>If this matrix contains L where A = LL<sup>\u22a4</sup> before the update, it will contain L
+   * where LL<sup>\u22a4</sup> = A + &sigma;vv<sup>\u22a4</sup> after the update.
```

Comment:
```suggestion
   * <p>If this matrix contains L where A = LLᵀ before the update, it will contain L
   * where LLᵀ = A + σvvᵀ after the update.
```

Reply:
Unfortunately, since viewing the PR info in GH shows the HTML parsed entity, it wasn't necessarily as clear that I was purposely swapping `&top;` with the Unicode escapes, as it is when viewing the commit log in the terminal. The Unicode escapes should ultimately render in the final HTML, but don't cause javadoc (in JDK 11) to fail. Still trying to see what the suggested change is using, since it's all post rendered characters on the web interface.

What led to the change was the failure for the _publish_ Gradle task, specifically _:docs:generateJavaDocs_ as below. I'm not sure about later JDK versions, because there appear to be bug reports specifically about supporting HTML5 entities, but those don't appear to have landed on JDK 11 builds, which WPIlib still uses (11.0.13 for on the last published release, and my personal version is 11.0.16.1).
```
> Task :docs:generateJavaDocs
/Users/brandonparsons/Developer/Code/allwpilib/wpimath/src/main/java/edu/wpi/first/math/Matrix.java:710: error: invalid entity &top;
   * <p>If this matrix contains L where A = LL<sup>&top;</sup> before the update, it will contain L
                                                   ^
/Users/brandonparsons/Developer/Code/allwpilib/wpimath/src/main/java/edu/wpi/first/math/Matrix.java:711: error: invalid entity &top;
   * where LL<sup>&top;</sup> = A + &sigma;vv<sup>&top;</sup> after the update.
                  ^
/Users/brandonparsons/Developer/Code/allwpilib/wpimath/src/main/java/edu/wpi/first/math/Matrix.java:711: error: invalid entity &top;
   * where LL<sup>&top;</sup> = A + &sigma;vv<sup>&top;</sup> after the update.
                                                  ^
3 errors

> Task :docs:generateJavaDocs FAILED

FAILURE: Build failed with an exception.
```

Reply:
Forgot to say that if the suggested change resolves the publish failure in a cleaner manner, I'm all for it.